docs/api/en/constants/Core.html

```diff
@@ -48,8 +51,15 @@ <h2>Mouse Buttons</h2>
 THREE.MOUSE.LEFT
 THREE.MOUSE.MIDDLE
 THREE.MOUSE.RIGHT
+THREE.MOUSE.ROTATE
+THREE.MOUSE.DOLLY
+THREE.MOUSE.PAN
 		</code>
-
+		<p>
+			The constants LEFT and ROTATE have the same underlying value.
+			The constants MIDDLE and ROTATE have the same underlying value.
```

Comment:
I think you meant DOLLY and PAN in these two lines.

Reply:
indeed